src/main/java/com/neighborhoodAssurance/NeighborController.java

```diff
@@ -21,21 +19,8 @@ public String index()
 	@ResponseBody
 	public String readJSON() 
 	{
-		/*

```

Comment:
I removed commented out code as it is bad practice to leave it in a master build.

Reply:
I agree.    Instead of commenting out code, my suggestion is:
1) Use feature branches in Git to keep experimental work separate from master.
2) If you're removing something that used to be there, just remove it entirely.  You can always go back in history to see what used to be there.